cpp/src/slice2cs/Main.cpp

```diff
@@ -23,6 +23,44 @@ namespace
     bool interrupted = false;
 }
 
+class CsharpDocCommentFormatter final : public DocCommentFormatter
+{
+    void preprocess(StringList& rawComment) final
```

Comment:
C# uses XML tags in it's doc-comments, so we need to escape any XML special characters before parsing the doc-comment.